cpg-analysis/src/test/kotlin/de/fraunhofer/aisec/cpg/analysis/fsm/ComplexDFAOrderEvaluationTest.kt

```diff
@@ -409,6 +411,41 @@ class ComplexDFAOrderEvaluationTest {
         assertFalse(everythingOk, "Expected incorrect order")
     }
 
+    @Test
+    fun testSuccessWhileLoop2FSM() {
+        val functionOk =
```

Comment:
You can use the `byNameOrNull` function to find it easier, it works on all `DeclarationHolder` interfaces.
```kotlin
var complex = tu.byNameOrNull<RecordDeclaration>("ComplexOrder")
assertNotNull(complex)

var functionOk = complex.byNameOrNull<FunctionDeclaration>("okWhile2")
```

or even just

```kotlin
var functionOk = tu.byNameOrNull<RecordDeclaration>("ComplexOrder")?
                   .byNameOrNull<FunctionDeclaration>("okWhile2")
```